app/renderer/api/shortcuts.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const Mousetrap = require('mousetrap');
+
+const AppStatus = require('../app_status');
+const GauchoActions = require('./gaucho_actions');
+const Material = require('./materialize');
+
+// Methods interface with jQuery
+module.exports = {
+    init() {
+        this.changeTabForward();
+        this.changeTabBack();
+        this.toggleEdit();
+        this.toggleNavBarMenu();
+    },
+    changeTabForward() {
+        Mousetrap.bind('ctrl+tab', () => {
+            const totalTabs = Material.getLengthNavBarTabs() - 1
+            const newIndex = AppStatus.activeSuite === totalTabs
+                ? 0
+                : AppStatus.activeSuite + 1
+
+            Material.selectTab("#navbar-tabs", `tab${newIndex}`);
+            GauchoActions.toggleActiveSuite(newIndex);
```

Comment:
I see a lot of repeated code in this method and changeTabForward

maybe part of the logic could be extracted into a third common method?